dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/WECC/Parameters.mo

```diff
@@ -105,4 +105,11 @@ record Parameters "Parameters of the PV WECC model"
   annotation(preferredView = "text");
   end Params_VSourceRef;
 
+  record Params_DriveTrain
+    parameter Types.Time Ht "Turbine Inertia in seconds (typical: 5s)";
+    parameter Types.Time Hg "Generator Inertia in seconds (typical: 1s)";
+    parameter Types.PerUnit Dshaft "Damping coefficient in p.u. (typical: 1.5 p.u.)";
```

Comment:
base

Reply:
is Dshaft influenced by the base here?

Reply:
Yes, it should be the same p.u base as the other variables it is mixed with in the equations of the model (power or torque here I think).

Reply:
For the units, the equations are homogeneous to : 
For Kshaft: Power = Kshaft dw/dt * w 
For Dshaft : Power = D * w^2
Hence, if we divide by Snom for P and omegaNom for w, we have a per unit base of Snom/omegaNom^2
Do you agree with this?

Reply:
In that case we write "in p.u (base SNom, omegaNom)", thanks !